grails-app/services/au/org/emii/portal/BulkDownloadService.groovy

```diff
@@ -145,4 +145,14 @@ class BulkDownloadService {
         zipStream.write bytes, 0, bytes.length
         zipStream.closeEntry()
     }
+
+    public static void copy(final InputStream input, final OutputStream output, final int bufferSize) throws IOException
```

Comment:
Ok, I understand.  Don't worry about it too much, I know what a PITA it can be to get grails dependencies to play nicely together.